contracts/evm/src/dvn/utils/AddressCast.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: LZBL-1.2
+
+pragma solidity ^0.8.20;
+
+library AddressCast {
+    error AddressCast_InvalidSizeForAddress();
+    error AddressCast_InvalidAddress();
+
+    function toBytes32(
+        bytes calldata _addressBytes
+    ) internal pure returns (bytes32 result) {
```

Comment:
A lot of protocols change the way they approach transformations between global id's and otherwise, are we making large assumptions here? It's okay for nuff because we just truncate.